lib/Zonemaster/CLI.pm

```diff
@@ -358,9 +356,13 @@ sub run {
     }
 
     # These two must come after any profile from command line has been loaded
-    # to override the profile setting
-    Zonemaster::Engine::Profile->effective->set( q{net.ipv4}, 0+$self->ipv4 );
-    Zonemaster::Engine::Profile->effective->set( q{net.ipv6}, 0+$self->ipv6 );
+    # to make any IPv4/IPv6 option override the profile setting.
+    if defined ($self->ipv4) {
+        Zonemaster::Engine::Profile->effective->set( q{net.ipv4}, 0+$self->ipv4 );
+    }
+    if defined ($self->ipv6) {
+        Zonemaster::Engine::Profile->effective->set( q{net.ipv6}, 0+$self->ipv6 );
+    }
```

Comment:
I tested it, and it looks like the parentheses usage in the if statement makes perl comply.

Reply:
I should have kept the first version... Fix in a minute.